pkg/reconciler/cronjobsource/controller.go

```diff
@@ -69,6 +71,7 @@ func NewController(
 		deploymentLister: deploymentInformer.Lister(),
 		eventTypeLister:  eventTypeInformer.Lister(),
 		env:              *env,
+		context:          ctx,
```

Comment:
rename to `loggingContext`, as with apiserversource?

Reply:
sure, done